bin/node/cli/benches/transaction_pool.rs

```diff
@@ -84,6 +84,10 @@ fn new_node(tokio_handle: Handle) -> node_cli::service::NewFullBase {
 		rpc_cors: None,
 		rpc_methods: Default::default(),
 		rpc_max_payload: None,
+		rpc_max_request_size: None,
+		rpc_max_response_size: None,
+		rpc_id_provider: None,
+		rpc_max_subs_per_conn: None,
```

Comment:
@dvdplm bikeshed on the naming here:

shall we reanme `rpc_id_provider` and `rpc_max_subs_per_conn` to ``rpc_ws_id_provider` and `rpc_ws_max_subs_per_conn`? As it is for the `rpc_ws_max_connections`?

Reply:
I think not. There's the webrtc transport coming down the pipe at some point so it's better to not equate "has subscriptions" with "websocket transport". The `rpc_ws_max_connections` should perhaps have been called `rpc_max_connections` but then again I'm not sure if connections work the same way in webrtc (or any other future stateful transport).

Reply:
yepp, I was thinking about future proof the CLI options e.g. some other stateful transport.

so would be better rename `rpc_ws_max_connections` to `rpc_max_connections` to be consistent but let's not do it in this PR.